tool_collections/fastx_toolkit/fasta_clipping_histogram/fasta_clipping_histogram.xml

```diff
@@ -1,18 +1,28 @@
-<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="1.0.0">
+<tool id="cshl_fasta_clipping_histogram" name="Length Distribution" version="@VERSION@">
     <description>chart</description>
-    <requirements>
-        <requirement type="package" version="0.0.13">fastx_toolkit</requirement>
-    </requirements>
-    <command>fasta_clipping_histogram.pl $input $outfile</command>
-
+    <expand macro="requirements" />
+    <macros>
+        <import>fastx_macros.xml</import>
+    </macros>
+    <macros>
+        <import>fastx_macros.xml</import>
+    </macros>
+    <command><![CDATA[
+        command -v perl5.22.0 &&
+        perl5.22.0 `command -v fasta_clipping_histogram.pl` '$input' '$outfile' ||
+        fasta_clipping_histogram.pl '$input' '$outfile'
+    ]]></command>
     <inputs>
         <param format="fasta" name="input" type="data" label="Library to analyze" />
     </inputs>
-
     <outputs>
         <data format="png" name="outfile" metadata_source="input" />
     </outputs>
     <tests>
+        <test>
```

Comment:
@bgruening unfortunately this test only passes if I set galaxy's conda version to 4.2.7 :(.
Otherwise I'm getting fastx_toolkit==0.0.14-2 instead of -3. See https://travis-ci.org/galaxyproject/tools-devteam/jobs/160679791#L355